code/game/gamemodes/revolution/revolution.dm

```diff
@@ -5,6 +5,7 @@
 	extended_round_description = "Revolutionaries - Remove the heads of staff from power. Convert other crewmembers to your cause using the 'Convert Bourgeoise' verb. Protect your leaders."
 	required_players = 4
 	required_enemies = 3
+	ert_disabled = 1

```

Comment:
That is not noted in the PR description.
Do not try to sneak in controversial changes that are not noted in the PR description.

Reply:
i suppose i should've marked that this is pretty much a work in progress
regardless, it's in the changelog, but i'll edit the pr description